src/WebJobs.Script.Grpc/Abstractions/WorkerDescription.cs

```diff
@@ -30,7 +30,7 @@ public class WorkerDescription
         /// <summary>
         /// Gets or sets the default path to the worker (relative to the bin/workers/{language} directory)
         /// </summary>
-        [JsonProperty(PropertyName = "defaultWorkerPath")]
+        [JsonProperty(PropertyName = "defaultWorkerPath", Required = Required.AllowNull)]
```

Comment:
Remove required. By default is to allow nulls. Any of these properties are optional as they can be overridden  by a profile. Here is an example: [Java worker.config.json](https://github.com/Azure/azure-functions-java-worker/blob/dev/worker.config.json#L10)